docs/reference/method/MongoDBClient-selectCollection.txt

```diff
@@ -72,7 +72,7 @@ with a custom read preference:
        'test',
        'users',
        [
-           'readPreference' => new MongoDB\Driver\ReadPreference('primaryPreferred'),
+           'readPreference' => new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::PRIMARY_PREFERRED),
```

Comment:
I don't feel strongly about this as the strings values are often used as-is in connection strings (similar to "majority" for write concerns). This may makes the example code harder to read in the docs UI since there's no wrapping, but we already have that problem for method signatures.

That said, the UI issue is a separate issue we'll need to address down the line (such as guidelines on wrapping and splitting method declarations across multiple lines for readability).

Reply:
The other way was done in https://github.com/mongodb/mongo-php-library/commit/dea346e48c8b8b44ae193371b3e225a3a979274e#diff-f631b3b3beb6be1d2034cda96ac452537b5827b20a04c64b45c86179ec7c29b8R75

This would be more concise with class imports (`use` statements), and that's how developers write their code in the end. But that's not how the doc is written.

Reply:
Aye, we tend to avoid `use` statements in the docs since they're often snippets and not complete PHP files. The flip side of that is needing to refer to things by their FQCNs all the time.

If you're OK to I think that's best. I think the users that care will find the constants and use them, but readability is in the best interest of our docs audience.

Reply:
Reverted.